internal/rest/dequeue_nodes.go

```diff
@@ -410,13 +410,7 @@ func (rest *RestOperations) ExecuteRestAndPopulateCache(rest_ops []*utils.RestOp
 						publishKey = splitKeys[1]
 					}
 				}
-				_, validAviError := err.(session.AviError)
-				if !validAviError && avimodel != nil && avimodel.GetRetryCounter() != 0 {
-					// We may face this error if the connection is idle for some time
-					utils.AviLog.Errorf("key: %s, error in parsing the error to avi error: %v, adding to fast retry queue", key, err.Error())
-					rest.PublishKeyToRetryLayer(publishKey, aviclient, key)
-					return false
-				}
+
```

Comment:
Let's say AKO is idle, someone creates and ingress. The session would have expired. We would wait for 2 mins before the ingress syncs.